src/qautostart.cpp

```diff
@@ -31,102 +31,86 @@
 #include <QFile>
 #include <QDir>
 
-
-Autostart::Autostart() {
-
-}
-
 #if defined (Q_OS_WIN)
 #define REG_KEY "HKEY_CURRENT_USER\\Software\\Microsoft\\Windows\\CurrentVersion\\Run"
 
-bool Autostart::isAutostart() const {
+bool Autostart::isAutostart() const
```

Comment:
Nothing has changed here. Please follow this [code convention](https://google.github.io/styleguide/cppguide.html) to minimize the changes.

Reply:
Yes, but that's just my style. I think it looks better. I don't understand why changes should be minimized (please don't be hard on me, I don't know a lot about GitHub).